index.html

```diff
@@ -0,0 +1,53 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+  <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+  <link href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,700;1,300;1,500&display=swap" rel="stylesheet">
+  <!-- Material Icons -->
+    <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet" />
+    <link rel="stylesheet" href="style.css">
+    <title>Document</title>
+</head>
+<body>
+  
+  <main class="main">
+ 
+<header class="nav-bar">    
+  <div class="logo-left">My Logo</div>
+ <i class="material-icons">menu</i>
+</header>
+
+<section class="headline">
+   <h1>I'm Cookey<br>Glad to see <br> you!</h1>
+    <p>I'm a software developer! I can help you build a product, feature or website. Look through some experience! If you like what you see and have a project you need coded, don't hesitate to contact me.
+    </p>
+
```

Comment:
- [ ] Your paragraph content is not matching the Figma design so make it similar to it. use the same font-family and color for it
### your design
![hispara](https://user-images.githubusercontent.com/81727988/170711619-13cfb96b-ba64-47ac-994e-a94c72fc5058.png)
### Figma design
![figmapara](https://user-images.githubusercontent.com/81727988/170711660-0b13c0bf-c37c-4748-9efe-539e41515361.png)